config.json

```diff
@@ -1,86 +1,189 @@
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/loadBalancer.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/network.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/networkInterface.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/networkWatcher.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/publicIpAddress.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/routeFilter.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/routeTable.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/serviceCommunity.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/routeTable.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/routeTable.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/usage.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/virtualNetwork.json",
+              "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/virtualNetworkGateway.json"],
+            "namespace": "Azure::ARM::Network::Api_2017_09_01",
+            "package-version": "0.11.0"
+        },
+        "azure_mgmt_network_2017_03_30":{
+            "input-file": ["https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/vmssNetworkInterface.json",
+                            "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/network/resource-manager/Microsoft.Network/2017-09-01/vmssPublicIpAddress.json"],
+            "namespace": "Azure::ARM::Network::Api_2017_03_30",
+            "package-version": "0.11.0"
+        }//,
+        // "azure_mgmt_network_2017_08_01":{
```

Comment:
I'm not sure how we're going to package network (that's one of my pending questions - since there are 2 files in the packages defined in readme.md with different versions, and they actually belong to compute - see https://github.com/Azure/azure-rest-api-specs/issues/1685), if one of the sets looks correct, i'll add the rest.